<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>routeUi</class>
 <widget class="QWidget" name="routeUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>33</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSpinBox" name="penSizeSB">
     <property name="toolTip">
      <string>Modify the width of the drawn route line</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="lineStyleCB">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select line style of route</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="routeColorPB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select route color</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="vehicleCB">
     <property name="toolTip">
      <string>Select vehicle for the route</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="vehicleSettingsPB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Vehicle Settings</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="routegen.qrc">
       <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="totalTimeCB">
     <property name="toolTip">
      <string>Turns on/off automatic route interpolation</string>
     </property>
     <property name="text">
      <string>Set total time (s) :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="routeTimeSB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Length of generated route in seconds</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="smoothPathCB">
     <property name="text">
      <string>Smooth route</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="routegen.qrc"/>
 </resources>
 <connections/>
</ui>
